client/vscode/src/inlineCompletionActivation.ts

```diff
@@ -62,9 +99,21 @@ class CodeWhispererInlineCompletionItemProvider implements InlineCompletionItemP
             token
         )
 
-        const list: InlineCompletionList = response as InlineCompletionList
+        const list: InlineCompletionListWithReferences = response as InlineCompletionListWithReferences
         this.languageClient.info(`Client: Received ${list.items.length} suggestions`)
 
-        return list
+        console.log('Got inlineCompletionsWithReferences from server', list)
+
+        const firstCompletionDisplayLatency = Date.now() - requestStartTime
+        // Add completion session tracking and attach onAcceptance command to each item to record used decision
+        list.items.forEach((item: InlineCompletionItemWithReferences) => {
```

Comment:
If we see all of items or a few of them, and then don't select any - does client still call `LogInlineCompletionSessionResults` with some as seen or discard?

Reply:
In this implementation I don't track visibility or discard (`seen` flag). In language server we mark those as `Discard` by default https://github.com/aws/language-servers/blob/main/server/aws-lsp-codewhisperer/src/language-server/session/sessionManager.ts#L106

Reply:
This can be approved in follow-up, but I wanted to have some minimal functionality working

Reply:
makes sense, this will definitely help in UTDE testing end-to-end.